CONTRIBUTING.md

```diff
@@ -461,6 +461,10 @@ not needed to use `sbt scalafmtAll` when editing with IntelliJ.
 
 PR validation includes checking that the Scala sources are formatted and will fail if they are not.
 
+Akka prefer the flatten imports rather than grouped, which helps reducing merge conflicts. If you are using IDEA, you can set 
+the option with: `Preferences` -> `Code Style` -> `Scala` -> `Imports` -> `Collect Imports with the same prefix into one import`.
```

Comment:
Should that be disable/unset, because we do not want to collect all imports into one?